nsqadmin/nsqadmin.go

```diff
@@ -115,7 +115,7 @@ func (n *NSQAdmin) Main() {
 	n.Unlock()
 	httpServer := NewHTTPServer(&Context{n})
 	n.waitGroup.Wrap(func() {
-		http_api.Serve(n.httpListener, httpServer, "HTTP", n.opts.Logger)
+		http_api.Serve(n.httpListener, http_api.CompressHandler(httpServer), "HTTP", n.opts.Logger)
```

Comment:
In an ideal world we'd have _one_ type of middleware decorator.  I realize you probably did this so that we didn't have to specify it exhaustively per-endpoint, but really we only need it for the two javascript payloads, all the other ones are negligible.

Thoughts?